src/Common/src/Interop/Windows/mincore/Interop.GetConsoleTitle.cs

```diff
@@ -20,13 +20,22 @@ internal partial class mincore
         // throw.
         private const int MaxAllowedBufferSizeInChars = 24500;
 
+        // GetConsoleTitle sometimes interprets the second parameter (nSize) as number of bytes and sometimes as the number of chars.
```

Comment:
Do we have a bug around SetConsoleTitle as well?  I'm wondering if it does a similar interpretation.